### PR TITLE
LMS Page

### PR DESCRIPTION
**Pull request by nicolasabousawan**
This is a place holder route the will contain sample videos that the cleaner can access.

- [x] Access via /learn 
- [ ] Format/CSS is as per standards (2+ SEs must agree)
- [x] placeholder content is easily replaceable with new content (technical requirement)

### CODE REVIEW

File: ui/src/Components/Learn.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export default class ProfilePage extends React.Component {
+    constructor(props){
+      super(props);
+    }
+  
+    render(){
+      return(
+        <Table className="row-buffer" responsive>
+            <h3> Start Learning Here! </h3>
```
Comment (gurleen-sanghera) [inline ui/src/Components/Learn.jsx]: I think you need to add some customized CSS. All other pages are more cohesive in terms of the titles and text. 
image
image

^ This shows the titles and text on other pages and how they follow the same styling vs the below shows the titles/text in the LMS

image
image

**Review by gurleen-sanghera — CHANGES_REQUESTED** [review]
- Access via /learn: this looks good, I like the rerouting back to home too :)
-  Format/CSS is as per standards (2+ SEs must agree): this is a bit inconsistent, I left comments with pictures
 - placeholder content is easily replaceable with new content (technical requirement): I read up on iFrames and looks like removing the link + certain addition code you added (allowing for fullscreen, etc.) and replacing with new content is very doable. Since we don't know how they want this to be formatted though, I don't think this point is too much of an issue.

**Comment by nicolasabousawan** [comment]
Yes @gurleen-sanghera good call out, forgot to push my changes. The Title now matches the same font as all title pages. The captions now match the same font as the labels of the form (i.e first name in /create) 
If we both sign off on this and get @kvptkr to approve the design, we should be good to merge.

**Comment by kvptkr** [comment]
@nicolasabousawan could you just make sure you deploy it and things are working and then I can approve things 😄

**Review by gurleen-sanghera — COMMENTED** [review]
Looks good Nicolas!

**Review by gurleen-sanghera — APPROVED** [review]

**Pull request merged**